kiwi/builder/disk.py

```diff
@@ -206,8 +206,15 @@ def __init__(self, xml_state, target_dir, root_dir):
         self.disk_setup = DiskSetup(
             xml_state, root_dir
         )
+        self.custom_args = custom_args
+
+        self.boot_signing_keys = None
+        if custom_args and 'signing_keys' in custom_args:
+            self.boot_signing_keys = custom_args['signing_keys']
+
         self.boot_image = BootImage(
-            xml_state, target_dir, root_dir
+            xml_state, target_dir, root_dir,
+            signing_keys=self.boot_signing_keys
```

Comment:
Ok, agree with the variable name change. Note here I have not used the key=value style as the `root_dir` parameter is also provided.